src/main/java/com/spartronics4915/frc2019/subsystems/CargoIntake.java

```diff
@@ -95,8 +96,8 @@ public void onLoop(double timestamp)
                     case HOLDING:
                         if (mStateChanged)
                         {
-                            mMotor1.set(ControlMode.PercentOutput, 0);
-                            mMotor2.set(ControlMode.PercentOutput, 0);
+                            mMotorRight.set(ControlMode.PercentOutput, 0);
```

Comment:
For all states, the motor sets should be outside of the if(stateChanged) block. The motor calls should run every loop, or else there will be an error.